packages/vertica-nodejs/mochatest/integration/client/copy-tests.js

```diff
@@ -0,0 +1,121 @@
+    pool.query("COPY copyTable FROM LOCAL 'copy-good.dat' RETURNREJECTED", (err, res) => {
+      assert.equal(res.rows[0]['Rows Loaded'], 5) // 5 good rows in goodFileContents
+      assert.equal(err, undefined)
+      assert.deepEqual(res.getRejectedRows(), [])
+      done()
+    })
+  })
+
+  it('returns rejected rows with RETURNREJECTED specified', function(done) {
+    pool.query("COPY copyTable FROM LOCAL 'copy-bad.dat' RETURNREJECTED", (err, res) => {
+      assert.equal(res.rows[0]['Rows Loaded'], 3) // 3 good rows in badFileContents
+      assert.equal(err, undefined)
+      assert.deepEqual(res.getRejectedRows(), [2, 4]) // rows 2 and 4 are malformed
+    })
+    done()
+  })
+
+  it('writes rejects to file with REJECTED DATA specified', function (done) {
+    pool.query("COPY copyTable FROM LOCAL 'copy-bad.dat' REJECTED DATA 'rejects.txt'", (err, res) => {
+      assert.equal(res.rows[0]['Rows Loaded'], 3) // 3 good rows in badFileContents
+      assert.equal(err, undefined)
+      fs.readFile('rejects.txt', 'utf8', (err, data) => {
+        assert.equal(err, undefined)
+        assert.equal(data, "'b'|2\n'd'|4\n") // rows 2 and 4 are malformed
+        fs.unlink('rejects.txt', done)
+      })
+    })
+  })
+
+  it('succeeds when data file is larger than buffer size requiring multiple copyData messages', function(done) {
+    const largeFilePath = path.join(process.cwd(), "large-copy.dat")
+    const writableStream = fs.createWriteStream(largeFilePath, { encoding: 'utf8' });
+    const bytesPerLine = 6 // single quote + letter + single quote + bar + integer + newline = 6 bytes
+    const desiredFileSize = 66000 // 65536 is our max buffer size. This will force multiple copyData messages
+    const requiredLines = desiredFileSize / bytesPerLine
+
+    for (let i = 1; i <= requiredLines; i++) {
+      const char = String.fromCharCode('a'.charCodeAt(0) + (i % 26)); // a - z
+      const line = `${i}|'${char}'\n`
+      writableStream.write(line)
+    }
+    writableStream.end()
+
+    pool.query("COPY copyTable FROM LOCAL 'large-copy.dat' RETURNREJECTED", (err, res) => {
+      assert.equal(res.rows[0]['Rows Loaded'], requiredLines)
+      assert.equal(err, undefined)
+      assert.deepEqual(res.getRejectedRows(), [])
+      fs.unlink(largeFilePath, done)
+    })
+  })
+
+  it('succeeds with binary copy local files', function(done) {
+    const binaryFileContents = Buffer.from(goodFileContents, 'utf-8')
+    const binaryFilePath = path.join(process.cwd(), 'binary-copy.bin')
+    fs.writeFile(binaryFilePath, binaryFileContents, () => {
+      pool.query("COPY copyTable FROM LOCAL 'binary-copy.bin' RETURNREJECTED", (err, res) => {
+        assert.equal(res.rows[0]['Rows Loaded'], 5)
+        assert.equal(err, undefined)
+        assert.deepEqual(res.getRejectedRows(), [])
+        fs.unlink(binaryFilePath, done)
+      })    
+    })
+  })
+
+  
+  it('succeeds with multiple copy local files', function(done) {
+    done()
```

Comment:
Are these empty tests?

Reply:
I included some placeholders from the test plan that included copy from standard in as well so some tests at the end are empty